operators/understanding/olm/olm-operatorconditions.adoc

```diff
@@ -17,3 +17,5 @@ include::modules/olm-supported-operatorconditions.adoc[leveloffset=+1]
 
 * xref:../../../operators/admin/olm-managing-operatorconditions.adoc#olm-operatorconditions[Managing Operator conditions]
 * xref:../../../operators/operator_sdk/osdk-generating-csvs.adoc#osdk-operatorconditions_osdk-generating-csvs[Enabling Operator conditions]
+* xref:../../../nodes/nodes/nodes-nodes-graceful-shutdown.adoc#nodes-nodes-graceful-shutdown[Managing graceful node shutdown]
+* xref:../../../applications/deployments/route-based-deployment-strategies.adoc#deployments-graceful-termination_route-based-deployment-strategies[Graceful termination]
```

Comment:
Possibly also worth a link to docs for PodDisruptionBudgets like [these][1]?

[1]: https://docs.openshift.com/container-platform/4.13/nodes/pods/nodes-pods-configuring.html#nodes-pods-pod-distruption-about_nodes-pods-configuring

Reply:
Updated. Thank you, @wking! I added the reference. I also had to remove the reference to "Managing graceful node shutdown", as the content is being removed from 4.13 and 4.14.